colin/cli/colin.py

```diff
@@ -85,7 +85,7 @@ def check(target, ruleset, ruleset_file, debug, json, stat, tag, verbose):
         if not results.ok:
             sys.exit(1)
         elif results.fail:
-            sys.exit(2)
+            sys.exit(3)
```

Comment:
@lachmanfrantisek What about to add there a comment why exit(2) is not used? I would prefer to comment all exit codes. Does it make sense to you?

Reply:
Isn't the README enough?

Reply:
Yeah, let's leave it, just digging.